src/Parser/Hunter/Survival/Modules/Abilities.js

```diff
@@ -216,6 +232,30 @@ class Abilities extends CoreAbilities {
           base: 1500,
         },
       },
+      {
+        spell: [SPELLS.CALL_PET_1, SPELLS.CALL_PET_2, SPELLS.CALL_PET_3, SPELLS.CALL_PET_4, SPELLS.CALL_PET_5],
+        category: Abilities.SPELL_CATEGORIES.UTILITY,
+        gcd: {
+          base: 1500,
+        },
+      },
+
+      /**
+       * Following are implemented to avoid annoying console messages.
+       */
+
+      {
+        spell: SPELLS.DEFILED_AUGMENT_RUNE,
+        category: Abilities.SPELL_CATEGORIES.HIDDEN,
+      },
+      {
+        spell: SPELLS.POTION_OF_PROLONGED_POWER,
```

Comment:
I plan to make a module for pots so you can leave these out if you want (or you can leave them in until I have added the pots globally, up to you).